src/components/Checkbox/Checkbox.tsx

```diff
@@ -65,4 +69,10 @@ const Checkbox: FC<CheckboxProps> = props => {
     );
 };
 
+Checkbox.defaultProps = {
+    textVerticalAlign: 'top',
```

Comment:
See my comment on the PR why I consider this a breaking change. @duranmla do you agree with this?

Reply:
Got it, I understand your point. But it almost feels like we need to comment on the "breaking" change on the documentation instead, isn't it? As you are adding a custom element we can't guarantee you won't break things. Maybe we can keep the default as `top` because also the default user is not using custom labels.

Besides, I think we should strive for making pieces that own their space without affecting the exterior. A headline maybe should not have an inherent margin?

Reply:
My point above is related to https://youtu.be/xxbc3wAztl0?t=551 _" components shouldn't contain surrounding white space"_ in that sense I wouldn't shape a component to meet the opposite of the quote I just give to you here.

Reply:
I agree that the default behavior SHOULD be `top`, but supporting custom elements was a use case we explicitly wanted and do support currently. Headline is just an example but more common is to use `Checkbox` with an icon and text combination for example.

The `top` option only works in very limited situation so with the current implementation of `Checkbox` I don't think we should make this default.

I definitely think this is something we should fix in the next major version though.

> Besides, I think we should strive for making pieces that own their space without affecting the exterior. A headline maybe should not have an inherent margin?

The problem is not the margin. The checkbox should be centered on the first line of the text but sticks to the top. So when the checkbox is significantly smaller or bigger than line-height he label, it will look broken.

Reply:
So to avoid breaking changes shall we revert to have the default to be `center`?

Reply:
Got it @lukahartwig . Yes, @div-Leo let's keep the default behavior to `center` then.

Reply:
cool from my side :)